advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,586 @@
+...
+P00  DEBUG:     storage/storage::storageNewWrite: => {
+	type: posix, name: {"/var/lib/edb/as13/backups/repo2/archive/demo/13-1/0000000100000000/
+							000000010000000000000005-be9a61a800934879842fa647b0d50385f44e0228.gz"}, 
+	modeFile: 0640, modePath: 0750, createPath: true, syncFile: true, syncPath: true, atomic: true}
+...
+P00   INFO: pushed WAL file '000000010000000000000005' to the archive
+```
+
+The `archive-push` command will try to push the WAL archive to all reachable repositories. The idea is to archive to as many repositories as possible even if we still need to throw an error to PostgreSQL to prevent it from removing the WAL file.
+
+```
+P00   INFO: archive-push command begin: [pg_wal/000000010000000000000006] ...
+...
+P00   INFO: archive-push command begin: [pg_wal/000000010000000000000007] ...
+ERROR: [104]: archive-push command encountered error(s):
+       repo2: [PathOpenError] unable to list file info for path '...': [13] Permission denied
+P00   INFO: archive-push command end: aborted with exception [104]
+DETAIL:  The failed archive command was: pgbackrest --stanza=demo archive-push pg_wal/000000010000000000000007
+WARNING:  archiving write-ahead log file "000000010000000000000007" failed too many times, will try again later
+```
+
+The PostgreSQL `archiver` process should then report an error:
+
+```bash
+$ ps -o pid,cmd fx |grep archiver
+ 4440  \_ postgres: archiver failed on 000000010000000000000007
+```
+
+The next WAL segments shouldn't then be archived:
+
+```bash
+$ psql -d postgres -c "SELECT pg_create_restore_point('generate some activity'); SELECT pg_switch_wal();"
+ pg_switch_wal 
+---------------
+ 0/80001C8
+(1 row)
+
+$ ls as13/data/pg_wal/archive_status/ |grep ".ready"
+000000010000000000000007.ready
+000000010000000000000008.ready
+```
+
+This becomes very handy by adding `archive-async=y` to the configuration in order to use asynchronous archiving processes within pgBackRest itself. Even if PostgreSQL `archiver` process is still stuck, the archives will reach the working repositories:
+
+```bash
+$ $ ls /var/lib/edb/as13/backups/repo1/archive/demo/13-1/0000000100000000/
+000000010000000000000005-be9a61a800934879842fa647b0d50385f44e0228.gz
+000000010000000000000006-7108296955e1208c93447dcfc5712ce59af97907.gz
+000000010000000000000007-f1b6fcdd23e96a19a7a89dc92613674c2316aa49.gz
+000000010000000000000008-791c44d39ef772d58e51d64c8e3589231d3916a4.gz
+000000010000000000000009-47241214eb3fd5e693eed778caa7b304a778ecc2.gz
+
+$ ls /var/lib/edb/as13/backups/repo2/archive/demo/13-1/0000000100000000/
+000000010000000000000005-be9a61a800934879842fa647b0d50385f44e0228.gz
+000000010000000000000006-7108296955e1208c93447dcfc5712ce59af97907.gz
+```
+
+Let's unblock the `archiver` process (and remove asynchronous archiving) before going further:
+
+```bash
+$ $ ps -o pid,cmd fx |grep archiver
+ 4440  \_ postgres: archiver last was 00000001000000000000000
+```
+
+#### Backup Command
+
+Let's take some backups:
```

Comment:
replace 'some' with 'a'

Reply:
actually, I take 4 backups (2 full and 2 incr) in the example. So perhaps 'several' or 'a few' rather than 'some' ?

Reply:
I would suggest "a few"

Reply:
Let us take few backups:

Reply:
fixed in c2942c3. Is that good for you too @sebasmannem ?

Reply:
Yes